docs/config-description.md

```diff
@@ -1,452 +1,600 @@
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: Path to the SSL certificate file.
-### Key: ssl_key_file
+- **Description**: The path to the SSL certificate file.
+
+### ssl_key_file
+
 - **Required**: False
 - **Type**: string
 - **Default value**: None
 - **Constraints**: None
- -   **Description**: Path to the SSL key file.
-### Key: api_version.default
+- **Description**: The path to the SSL key file.
+
+### api_version.default
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 1
-- **Constraints**: The minimum value is `1`. The maximum value is `3`
- -   **Description**: Default API version Clio will run on.
-### Key: api_version.min
+- **Default value**: `1`
+- **Constraints**: The minimum value is `1`. The maximum value is `3`.
+- **Description**: The default API version that the Clio server will run on.
+
+### api_version.min
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 1
-- **Constraints**: The minimum value is `1`. The maximum value is `3`
- -   **Description**: Minimum API version.
-### Key: api_version.max
+- **Default value**: `1`
+- **Constraints**: The minimum value is `1`. The maximum value is `3`.
+- **Description**: The minimum API version to use.
+
+### api_version.max
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 3
-- **Constraints**: The minimum value is `1`. The maximum value is `3`
- -   **Description**: Maximum API version.
-### Key: migration.full_scan_threads
+- **Default value**: `3`
+- **Constraints**: The minimum value is `1`. The maximum value is `3`.
+- **Description**: The maximum API version to use.
+
+### migration.full_scan_threads
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 2
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: The number of threads used to scan the table.
-### Key: migration.full_scan_jobs
+- **Default value**: `2`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of threads used to scan the table.
+
+### migration.full_scan_jobs
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 4
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: The number of coroutines used to scan the table.
-### Key: migration.cursors_per_job
+- **Default value**: `4`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of coroutines used to scan the table.
+
+### migration.cursors_per_job
+
 - **Required**: True
 - **Type**: int
-- **Default value**: 100
-- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`
- -   **Description**: The number of cursors each coroutine will scan.
-
+- **Default value**: `100`
+- **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
+- **Description**: The number of cursors each coroutine will scan.
```

Comment:
Coroutine is an implementation detail - i'd suggest using "job" or some other generic word describing a running task